<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>1000</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>1000</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>1000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>900</width>
      <height>1000</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>900</width>
      <height>1000</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>900</width>
      <height>1000</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true">QTabBar::tab {
color: rgb(60, 60, 60);
height: 50px; 
width: 250px; 
background-color:rgb(135,135,135);
padding-left: 15px;
padding-right: 15px;
border-top: 5px solid rgb(255, 255,255);
}

QTabBar::tab:selected 
{
height: 50px; 
width: 250px;
color: white;
background: rgb(80,80,80);
font-weight: bold;
padding-left: 15px;
padding-right: 15px;
border-top: 5px solid rgb(0, 220, 0);
}

QTabBar::tab:hover{
background-color:  rgb(90,90,90);
color : qlineargradient(spread:pad, x1:0, y1:0.511364, x2:1, y2:0.523, stop:0 rgb(0, 220, 0), stop:1 rgb(0, 150, 0));
font-weight: bold;
}


QTabBar::scroller {width: 0px;}</string>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab_pos">
     <property name="minimumSize">
      <size>
       <width>900</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>900</width>
       <height>16777215</height>
      </size>
     </property>
     <attribute name="title">
      <string>주문(ORDER)</string>
     </attribute>
     <widget class="QWidget" name="verticalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>730</y>
        <width>841</width>
        <height>84</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item alignment="Qt::AlignHCenter">
        <widget class="Line" name="line">
         <property name="minimumSize">
          <size>
           <width>750</width>
           <height>3</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>750</width>
           <height>3</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout">
         <property name="leftMargin">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_2">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>70</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel
{
    color: black;
    background-color: #cacaca;
    border-width: 1px;
    border-color:white;
    border-style: solid;
    border-radius: 6;
 

}</string>
           </property>
           <property name="text">
            <string>입력내용</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QTextEdit" name="L2_text">
           <property name="minimumSize">
            <size>
             <width>500</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>500</width>
             <height>70</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="0" column="2">
          <widget class="QPushButton" name="L2_btn">
           <property name="minimumSize">
            <size>
             <width>100</width>
             <height>70</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>100</width>
             <height>70</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
    color: black;
    background-color:#ffcd4a;
    border-width: 1px;
    border-color:white;
    border-style: solid;
    border-radius: 6;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
	color: white
}
QComboBox:hover,QPushButton:hover
{
    border: 2px solid #ff9b50;
}</string>
           </property>
           <property name="text">
            <string>복사</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget">
      <property name="geometry">
       <rect>
        <x>1</x>
        <y>11</y>
        <width>841</width>
        <height>719</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QTabWidget" name="tabWidget_2">
         <property name="minimumSize">
          <size>
           <width>550</width>
           <height>700</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>550</width>
           <height>700</height>
          </size>
         </property>
         <property name="autoFillBackground">
          <bool>true</bool>
         </property>
         <property name="styleSheet">
          <string notr="true">QTabBar::tab {
    color: rgb(60, 60, 60);
    height: 60px; 
    width: 70px; 
    background-color:rgb(135,135,135);
    padding-left: 15px;
    padding-right: 15px;
    border-top: 5px solid rgb(255, 255,255);
}

QTabBar::tab:selected {
    height: 60px; 
    width: 70px;
    color: black;
    background: rgb(255,255,255);
    font-weight: bold;
    padding-left: 15px;
    padding-right: 15px;
    border-top: 5px solid #01DB01;
}

QTabBar::tab:hover {
    background-color:  rgb(90,90,90);
    color: qlineargradient(spread:pad, x1:0, y1:0.511364, x2:1, y2:0.523, stop:0 rgb(0, 220, 0), stop:1 rgb(0, 150, 0));
    font-weight: bold;
}

QTabBar::scroller {
    width: 0px;
}

QTabWidget::pane {
    color: rgb(220, 220, 220); /* 탭 바디의 배경색을 변경 */
}</string>
         </property>
         <property name="tabPosition">
          <enum>QTabWidget::North</enum>
         </property>
         <property name="tabShape">
          <enum>QTabWidget::Rounded</enum>
         </property>
         <property name="currentIndex">
          <number>0</number>
         </property>
         <property name="documentMode">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="tab_shrimp">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>1000000</width>
            <height>1000000</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <attribute name="title">
           <string/>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_squid">
          <attribute name="title">
           <string/>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_fish">
          <attribute name="title">
           <string/>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_meat">
          <attribute name="title">
           <string/>
          </attribute>
         </widget>
         <widget class="QWidget" name="tab_other">
          <attribute name="title">
           <string/>
          </attribute>
         </widget>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="minimumSize">
            <size>
             <width>280</width>
             <height>34</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>280</width>
             <height>34</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>14</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel
{
    color: black;
    background-color: #cacaca;
    border-width: 1px;
    border-color:white;
    border-style: solid;
    border-radius: 6;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}</string>
           </property>
           <property name="text">
            <string>선택내역</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="L1_table">
           <property name="minimumSize">
            <size>
             <width>275</width>
             <height>600</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>275</width>
             <height>600</height>
            </size>
           </property>
           <column>
            <property name="text">
             <string>주문메뉴</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>수량</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
           <column>
            <property name="text">
             <string>삭제</string>
            </property>
            <property name="textAlignment">
             <set>AlignCenter</set>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="L1_table_btn">
           <property name="minimumSize">
            <size>
             <width>280</width>
             <height>60</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>280</width>
             <height>60</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QPushButton
{
    color: black;
    background-color:#ffcd4a;
    border-width: 1px;
    border-color:white;
    border-style: solid;
    border-radius: 6;
    padding: 3px;
    padding-left: 5px;
    padding-right: 5px;
}

QPushButton:pressed
{
    background-color: QLinearGradient( x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #2d2d2d, stop: 0.1 #2b2b2b, stop: 0.5 #292929, stop: 0.9 #282828, stop: 1 #252525);
	color: white
}
QComboBox:hover,QPushButton:hover
{
    border: 2px solid #ff9b50;
}</string>
           </property>
           <property name="text">
            <string>완료</string>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QComboBox" name="comboBox">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>830</y>
        <width>94</width>
        <height>22</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>한국어</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>영어</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>태국</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>베트남</string>
       </property>
      </item>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_history">
     <attribute name="title">
      <string>주문기록(History)</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>595</x>
      <y>0</y>
      <width>175</width>
      <height>60</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>175</width>
      <height>60</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>175</width>
      <height>60</height>
     </size>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logo.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>메뉴</string>
    </property>
    <addaction name="actionCreat_Account"/>
    <addaction name="actionDelete_Account"/>
    <addaction name="separator"/>
    <addaction name="actionAdd_Menu"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCreat_Account">
   <property name="text">
    <string>Creat Account</string>
   </property>
  </action>
  <action name="actionDelete_Account">
   <property name="text">
    <string>Delete Account</string>
   </property>
  </action>
  <action name="actionAdd_Menu">
   <property name="text">
    <string>Edit Menu</string>
   </property>
  </action>
  <action name="actionDelete_Menu">
   <property name="text">
    <string>Delete Menu</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
